packages/data-models/src/homonym.js

```diff
@@ -65,13 +65,18 @@ class Homonym {
   }
 
   static readObject (jsonObject) {
+    console.info('Homonym readObject - ', jsonObject)
     let lexemes = [] // eslint-disable-line prefer-const
     if (jsonObject.lexemes) {
       for (const lexeme of jsonObject.lexemes) {
         lexemes.push(Lexeme.readObject(lexeme))
       }
+    } else {
+      const languageID = LMF.getLanguageIdFromCode(jsonObject.languageCode)
+      lexemes = [new Lexeme(new Lemma(jsonObject.targetWord, languageID), [])]
     }
-    const homonym = new Homonym(lexemes, jsonObject.form)
+    const homonym = new Homonym(lexemes, jsonObject.form || jsonObject.targetWord)
+    homonym.lemmasList = jsonObject.lemmasList
```

Comment:
we need this because we retrieve homonym data from remote as 
{ targetWord, lemmaList } without lexemes property

Reply:
can you add a comment that says this to the code?